docs/introduction/installation.rst

```diff
@@ -29,6 +43,45 @@ In your project's ``settings.py`` make sure you have all of::
 
 listed in ``INSTALLED_APPS``, *after* ``'cms'``.
 
+And make sure that your other settings conform to:
```

Comment:
Consider "And make sure that the following additions are made to your settings:" here